src/database/schema/ChannelListSchema.ts

```diff
@@ -398,6 +398,16 @@ class ChannelList implements BaseDbSchema {
     }
   };
 
+  static async updateChannelList(db: SQLiteDatabase, channelId: string, channelType: string) {
+    const query = `UPDATE channel_lists SET channel_type="${channelType}" WHERE id="${channelId}"`;
+    await db.executeSql(query);
+  }
+
+  static async deleteChannelList(db: SQLiteDatabase, channelId: string) {
+    const query = `DELETE FROM channel_lists WHERE id="topic_${channelId}"`;
+    await db.executeSql(query);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similarly, the SQL query in `deleteChannelList` is also vulnerable to SQL injection attacks. Use parameterized queries here as well.
```diff
-    const query = `DELETE FROM channel_lists WHERE id="topic_${channelId}"`;
-    await db.executeSql(query);
+    const query = 'DELETE FROM channel_lists WHERE id=?';
+    await db.executeSql(query, [`topic_${channelId}`]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->